utils/http_handler.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+
+class HttpHandler():
+    def __init__(self, url):
+        print("Setting up the url: " + url + "...")
+
+        if "http:" not in url:
+            url = "http://" + url
+        self.url = url
+        self.raw_response = self.make_http_request()
+
+    def make_http_request(self):
+        # TODO: Error handling
+        # TODO: Caching logic
+        # TODO: Retrying logic
+
+        print("\t Downloading website info...")
+        try:
+            return requests.get(self.url).text
+        except Exception:
```

Comment:
do you think that this obfuscates the error in the HttpHandler? I would expect that the caller of `HttpHandler.make_http_request` should handle the exception?